sympy/utilities/tests/test_lambdify.py

```diff
@@ -362,10 +362,17 @@ def test_numpy_old_matrix():
     assert isinstance(f(1, 2, 3), numpy.matrix)
 
 def test_python_div_zero_issue_11306():
+    # The Try:Except condition is put to check whether zero division returns answer or not.
     if not numpy:
         skip("numpy not installed.")
-    p = Piecewise((1 / x, y < -1), (x, y <= 1), (1 / x, True))
-    lambdify([x, y], p, modules='numpy')(0, 1)
+    p = Piecewise((1 / x, y < -1), (x, y < 1), (1 / x, True))
+    f = lambdify([x, y], p, modules='numpy')
+    numpy.seterr(all='raise')
```

Comment:
This particular example should always raise (see the original issue for details). So we should just `numpy.seterr(all='ignore')` for the test (and set it back to `raise` in a finally block after the test is done).